src/cascade/executor/dismodel_main.py

```diff
@@ -51,16 +55,26 @@ def main(args):
     execution_context = make_execution_context(gbd_round_id=6, num_processes=args.num_processes)
     plan = generate_plan(execution_context, args)
 
-    local_cache = LocalCache(maxsize=2)
+    local_cache = LocalCache(maxsize=200)
     for cascade_task_identifier in plan.cascade_jobs:
         cascade_job, this_location_work = plan.cascade_job(cascade_task_identifier)
         configure_execution_context(execution_context, args, this_location_work)
 
         if cascade_job == "bundle_setup":
             # Move bundle to next tier
             setup_tier_data(execution_context, this_location_work.data_access, this_location_work.parent_location_id)
-        elif cascade_job == "estimate_location":
-            estimate_location(execution_context, this_location_work, local_cache)
+        elif cascade_job == "estimate_location:prepare_data":
```

Comment:
This is a big list. Totally makes sense. Looks a little like an `eval()`, doesn't it?

Reply:
Yeah, this is an ugly sieve. I'm not sure exactly how to structure it more cleanly but you could clearly do better.

Reply:
You could make the arguments list homogeneous and then just do a dictionary look up to get the handler for a particular job name. Or have the reference to the handler as a parameter on the job, though that changes other things about the architecture.